Documentation/Storage-Configuration/NFS/nfs-csi-driver.md

```diff
@@ -207,3 +207,16 @@ To clean your cluster of the resources created by this example, run the followin
 ```console
 kubectl delete -f deploy/examples/csi/nfs/pvc-clone.yaml
 ```
+
+## Consuming NFS from an external source
+
+For consuming NFS services and exports external to the Kubernetes cluster (including those backed by an external standalone Ceph cluster), Rook recommends using Kubernetes' regular NFS consumption model.
```

Comment:
Is this mean Rook's recommendation to use kubernetes Native NFS for external cluster/source, not the CephCSI NFS Driver? in other words, for Internal ceph use cephCSI NFS and external cluster use kubernetes native NFS.

Reply:
That's correct.

The external mode can create the export directly and use k8s intree driver.

Reply:
And we can add the Cephcsi support later if it is required.

Reply:
out of curiosity what is the reason not to use nfs csi driver here?

Reply:
Yes we had two ways, the first one looked more direct.

1) So if the user already has the nfs exports, they can use this document to create the pv and pvcs https://docs.openshift.com/container-platform/4.14/storage/persistent_storage/persistent-storage-nfs.html
       Moreover if they don't have the exports created they can run the simple ceph cmd
      ceph nfs export create cephfs my-nfs /test myfs 
2. Secondly we can integrate this feature similar way we have other addons,
   Steps would be creating a new nfs-storage class from the python script and then ceph-csi will take care of the export and create the pv for us. (and need  update on the csi-users permissions)
   
From the thread https://github.com/rook/rook/pull/13876#discussion_r1578922881

Reply:
Moreover, it is a recommendation, 
And if we implement the code later as per the user's requirement we can change the flow and recommendations.

Reply:
@parth-gr Would the cephnfs csi driver work for external clusters? If so, it does seem preferred to recommend it for consistently with internal and external mode. If not, this sounds like a gap in the cephnfs csi driver.

Reply:
The CephNFS driver works best when someone wants to provision new NFS exports from an existing CephNFS. This requires privileges on Rook's side for creating those exports. So from an external standpoint where multi-tenancy may be involved, or where security is a focus, the non-Ceph case is the safer (and thus preferred) recommendation.

Reply:
Per separate discussion, how about adding a sentence such as:
```suggestion
For consuming NFS services and exports external to the Kubernetes cluster (including those backed by an external standalone Ceph cluster), Rook recommends using Kubernetes' regular NFS consumption model. This requires the Ceph admin to create the needed export, while reducing the privileges needed in the client cluster for the NFS volume.
```

Reply:
Updated